src/components/app-button-menu/AppButtonMenu.tsx

```diff
@@ -115,7 +115,7 @@ const AppButtonMenu = <T extends Pick<CategoryNameInterface, '_id'>>({
     </Box>
   )
 
-  const itemsLoad = !response.length && loading
+  const itemsLoad = !response?.length && isLoading
```

Comment:
It is incorrect condition. If you leave !response.length, if will be always in loading state, if there are no items in database

```suggestion
  const itemsLoad = !response || isLoading
```